src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/AncmIISExpressV2.wixproj

```diff
@@ -15,26 +14,20 @@
     <TempPlatform Condition="'$(Platform)' == 'x86'">Win32</TempPlatform>
     <TempPlatform Condition="'$(Platform)' == 'arm64'">ARM64</TempPlatform>
     <ToolsetInstallerNuspecFile>$(MSBuildProjectDirectory)\$(MSBuildProjectName).nuspec</ToolsetInstallerNuspecFile>
+    <!-- We import D.B.T ourselves in this file -->
+    <ImportDirectoryBuildTargets>false</ImportDirectoryBuildTargets>
+    <!-- False positive in aspnetcoremodulev2.wxs -->
+    <SuppressSpecificWarnings>5437</SuppressSpecificWarnings>
```

Comment:
You can probably remove this suppression if you change the directory definition in the .wxs file (I'll leave a comment in there as well)